electrophysiology/models.py

```diff
@@ -40,15 +42,23 @@ class ExtracellularRecording(BaseExperimentalData):
     impedances = models.ForeignKey(Dataset, blank=True, null=True, related_name="extracellular_impedances",
                                    help_text="binary array for measured impedance of each channel (ohms).")
     amplifier = models.ForeignKey(Amplifier, blank=True, null=True,
-                                  help_text="The amplifier used in this recording. TODO: multiple amplifiers?")
+                                  help_text="The amplifier used in this recording.")
+								   # TODO: multiple amplifiers? 
+								   # they would have their own ExtracellularRecording objects in that case.
```

Comment:
This is a difficult one. We have had this discussion lots of times in developing KwikFormat and phy.

Our options are:
1. Have a single .dat file with a custom .prb file for the "virtual probe" containing two probes going into one amplifier.
2. Have multiple .dat files, each with their own .prb file.

People will want to do both. Can we make that possible?